site/src/pages/docs/advanced-metrics.mdx

```diff
@@ -0,0 +1,73 @@
+# Collecting metrics
+
+Armeria has built-in support for collecting metrics both on the server and client side. This page describes how to enable this.
+
+The metric data is collected using the [Micrometer](https://micrometer.io/) library. Please consult its documentation for information on how to expose the collected metrics to various monitoring systems (JMX, Prometheus, etc)
+
+## Collecting server-side metrics (gRPC)
```

Comment:
If you don't mind, could you also add an example of an HTTP service that uses `MeterIdPrefixFunction`?

Reply:
Yep, I could give that a try. :+1: